docs/content/latest/sample-data/pgexercises.md

```diff
@@ -78,8 +78,9 @@ Connect to the `exercises` database.
 yugabyte=# \c exercises
 ```
 
-```
 You are now connected to database "exercises" as user "yugabyte".
+
+```output
```

Comment:
Line 81 is actually part of the output :)
````suggestion

```output
You are now connected to database "exercises" as user "yugabyte".
````